src/views/Submission.js

```diff
@@ -742,24 +742,24 @@ class Submission extends React.Component {
             <Button className='submission-button' variant={this.state.item.isUpvoted ? 'primary' : 'secondary'} aria-label={(this.state.item.isUpvoted ? 'Cancel like' : 'Like')} onClick={this.handleUpVoteOnClick}><FontAwesomeIcon icon='heart' /> {this.state.item.upvotesCount}</Button>
           </TooltipTrigger>
           <TooltipTrigger message='Submission link'>
-            <Button className='submission-button' variant='secondary' aria-label='Visit submission link' onClick={() => { window.open(this.state.item.contentUrl, '_blank') }}><FontAwesomeIcon icon={faLink} /></Button>
+            <Button className='submission-button' variant='secondary' aria-label='Visit submission link' onClick={() => { if (/^https?:\/\//i.test(this.state.item.contentUrl)) { window.open(this.state.item.contentUrl, '_blank', 'noopener,noreferrer') } }}><FontAwesomeIcon icon={faLink} /></Button>
```

Comment:
It seems like we might want to consider some type of function for this as it's done in a few different places.